storage/store.go

```diff
@@ -532,6 +532,7 @@ func (s *Store) Start(stopper *stop.Stopper) error {
 			// and initialize those groups.
 			return false, nil
 		}); err != nil {
+		s.mu.Unlock()
```

Comment:
Might it be better to turn the iterator into a sub function with a defer unlock at the start?